kv/rest.go

```diff
@@ -152,7 +153,7 @@ func (s *RESTServer) handleRangeAction(w http.ResponseWriter, r *http.Request, _
 	// A limit of zero implies no limit.
 	limit, err := strconv.ParseInt(r.FormValue(rangeParamLimit), 10, 64)
 	if len(r.FormValue(rangeParamLimit)) > 0 && err != nil {
-		http.Error(w, "error parsing limit: "+err.Error(), http.StatusBadRequest)
+		http.Error(w, fmt.Sprintf("error parsing limit: %s", err), http.StatusBadRequest)
```

Comment:
The `%s` format on `error` invokes `Error()`. I'm mildly indifferent to the change, but the two bits of code will always produce identical strings.